app/manifest/v2/_base.json

```diff
@@ -34,9 +34,7 @@
         "scripts/disable-console.js",
         "scripts/lockdown-install.js",
         "scripts/lockdown-run.js",
-        "scripts/lockdown-more.js",
-        "scripts/contentscript.js",
-        "scripts/inpage.js"
```

Comment:
notice that we are not injecting the inpage provider (nor related phishing redirect logic) in this branch, meaning that dapps can only connect to the wallet by bringing their own provider and establishing a connection over externally_connectable